src/Uplink/Config.php

```diff
@@ -27,15 +34,15 @@ class Config {
 	/**
 	 * Get the container.
 	 *
-	 * @return ContainerInterface
-	 *@throws \RuntimeException
-	 *
 	 * @since 1.0.0
 	 *
+	 * @throws RuntimeException
+	 *
+	 * @return ContainerInterface
 	 */
 	public static function get_container() {
 		if ( self::$container === null ) {
-			throw new \RuntimeException( 'You must provide a container via StellarWP\Uplink\Config::set_container() before attempting to fetch it.' );
+			throw new RuntimeException( 'You must provide a container via StellarWP\Uplink\Config::set_container() before attempting to fetch it.' );
```

Comment:
Oops. Not having exceptions in this class file translated before was an oversight.

```suggestion
			throw new RuntimeException( __( 'You must provide a container via StellarWP\Uplink\Config::set_container() before attempting to fetch it.', '%TEXTDOMAIN%' ) );
```